net-firewall/fwknop/fwknop-2.6.9-r1.ebuild

```diff
@@ -37,7 +37,6 @@ RDEPEND="${DEPEND}"
 
 REQUIRED_USE="
 	firewalld? ( server )
```

Comment:
I'd dare say a similar logic may apply to the most of the remaining flags requiring server, at least firewalld/iptables — considering that they have circular REQUIRED_USE.